Modules/_decimal/_decimal.c

```diff
@@ -548,10 +548,12 @@ getround(PyObject *v)
    initialized to new SignalDicts. Once a SignalDict is tied to
    a context, it cannot be deleted. */
 
+static mpd_context_t dflt_ctx;
+
 static int
 signaldict_init(PyObject *self, PyObject *args UNUSED, PyObject *kwds UNUSED)
 {
-    SdFlagAddr(self) = NULL;
+    SdFlagAddr(self) = &dflt_ctx.status;
```

Comment:
They will all share the same flags.

Will users start to misuse this construction method, then found surprise that its flags being changed without notice?

Reply:
Thanks! It seems that it is not easy to forbidding this directly, because `decimal.Context()` depends on the `signaldict` instance. If you have a good idea, please let me know :) Or let's wait for advice from other professionals~